Core/Battler.gd

```diff
@@ -146,17 +148,21 @@ func _handle_enemy_deaths() -> void:
 	for enemy: Enemy in _enemy_list:
 		enemy.get_party_component().set_party(_enemy_list)
 
-
-func _check_and_handle_battle_end() -> void:
+# return TRUE if battle have ended either with victory or defeat
```

Comment:
Seems good to me, if you could just make the comments above the functions as documentation comments by using ## instead of # that would be perfect :+1:

Reply:
Alright, will do